locale/en.po

```diff
@@ -5808,35 +5751,35 @@ msgstr ""
 "\n"
 "Each icon represents a unique environment that you can inhabit, with the panel on your right describing the conditions of the currently selected patch.\n"
 "\n"
-"If you select a patch connected to the one you occupy, which is highlighted with a flashing green border, you can press the \"Move to this patch\" button on the right to migrate there.\n"
+"If you select a patch connected to the one you occupy (which is highlighted with a flashing green border), you can press the \"Move to this patch\" button on the right to move there.\n"
 "\n"
 "Try selecting a patch to continue."
 
 msgid "TUTORIAL_MICROBE_EDITOR_REMOVE_ORGANELLE_TEXT"
 msgstr ""
 "Removing organelles also costs [b]MP[/b] [thrive:icon]MP[/thrive:icon] as it is a mutation to your species, [u]unless they were placed in the current editor session[/u]. \n"
 "\n"
-"You can right click organelles to bring up the [b]organelle menu[/b] and selecting the delete option to remove them. \n"
+"You can right click organelles to bring up the [b]organelle menu[/b] and select the delete option to remove them. \n"
 "\n"
 "If you make a mistake, you can undo any change you make in the editor.\n"
 "\n"
 "Click the undo button to continue."
 
 msgid "TUTORIAL_MICROBE_EDITOR_SELECT_ORGANELLE_TEXT"
 msgstr ""
-"When selecting parts in the menu, [u]pay close attention to the tooltips[/u] to understand what they do. Parts can do [u]more harm than good if you are not considerate of their function[/u].\n"
+"When selecting parts, [u]pay attention to the tooltips[/u] to understand what they do. Parts can do [u]more harm than good if you don't consider their function[/u].\n"
 "\n"
-"Each part requires maintenance (called [b]osmoregulation[/b] [thrive:icon]OsmoIcon[/thrive:icon]) and some run [b]processes[/b] that require certain compounds to function.\n"
+"Each part requires maintenance (called [b]osmoregulation[/b] [thrive:icon]OsmoIcon[/thrive:icon]) and some of them run [b]processes[/b] that require certain compounds to function.\n"
 "\n"
-"Additionally, the more parts you add the more compounds you will need to reproduce.\n"
+"The more parts you add, the more compounds you will need to reproduce.\n"
```

Comment:
I think that removing the "additionally" from here causes a problem with not separating this point from the previous one. Without it I think the interpretation might not be clear enough that this is more like "BTW when you add more parts reproduction takes longer" and not "more parts = more processes using compounds".

Reply:
I'll add that additionally back as I think it is important in marking the change of topic to the player.